### DISCUSSION TITLE
HA Integration blank

### DISCUSSION THREAD

**Discussion by teedubyeah** [briis/unifiprotect#476] [General]
Hello 
When trying to install or configure the integration the window is blank (see image). Can someone help?
Core
Version
core-2022.2.3
Supervisor
Version
supervisor-2022.01.1
Operating System
Home Assistant OS 7.2
Deployment
production

![image](https://user-images.githubusercontent.com/55287773/153241896-11833690-0e7d-4c2b-8f28-7b1507381b88.png)

**Comment by briis** [comment]
Most likely a browser caching issue. Try and do a hard refresh of your browser.

**Reply by teedubyeah** [reply]
I've tried Edge, Chrome and Vivaldi all are doing the same thing. Also tried Chrome on a different computer.

**Reply by briis** [reply]
Did you have it installed before 2022.2, so the Custom Integration?


Anyway, try and fill out the 3 empty text fields:

1. Field enter IP address of your UniFi Protect server
2. Field enter the local username 
3. Field enter password for local user.

**Reply by teedubyeah** [reply]
so I can fill out the info and get past it. Which I have now. but when I try to configure the integration I get this window. I have no clue what supposed to be there. 
![image](https://user-images.githubusercontent.com/55287773/153245385-371a2c41-6923-4eec-8958-8e3ede3271b3.png)

**Reply by teedubyeah** [reply]
This was installed after the 2022.2 upgrade.

**Reply by AngellusMortis** [reply]
![image](https://user-images.githubusercontent.com/490848/153245687-ddcfcf19-8cba-4364-8576-0e4954d8e075.png)
![image](https://user-images.githubusercontent.com/490848/153245711-3e6f98af-b0bc-46ed-b627-c8addea6e24f.png)

What language is your Home Assistant instance set to? Do any other integrations have this issue?

**Reply by teedubyeah** [reply]
My HA is in English and I've never had an issue like this.

**Reply by teedubyeah** [reply]
Here are some errors and logs associated with. 
![image](https://user-images.githubusercontent.com/55287773/153246668-a9510a22-d208-4877-a4b7-88b74cad3bde.png)

Logger: homeassistant.setup
Source: setup.py:289
First occurred: 11:29:43 AM (9 occurrences)
Last logged: 11:29:43 AM

Unable to prepare setup for platform unifiprotect.number: Platform not found (cannot import name 'ProtectRequiredKeysMixin' from 'custom_components.None.models' (/config/custom_components/None/models/__init__.py)).
Unable to prepare setup for platform unifiprotect.select: Platform not found (cannot import name 'ProtectRequiredKeysMixin' from 'custom_components.None.models' (/config/custom_components/None/models/__init__.py)).
Unable to prepare setup for platform unifiprotect.sensor: Platform not found (cannot import name 'ProtectRequiredKeysMixin' from 'custom_components.None.models' (/config/custom_components/None/models/__init__.py)).
Unable to prepare setup for platform unifiprotect.switch: Platform not found (cannot import name 'ProtectRequiredKeysMixin' from 'custom_components.None.models' (/config/custom_components/None/models/__init__.py)).
Unable to prepare setup for platform unifiprotect.camera: Platform not found (cannot import name 'ProtectRequiredKeysMixin' from 'custom_components.None.models' (/config/custom_components/None/models/__init__.py)).


Logger: homeassistant.helpers.integration_platform
Source: loader.py:583
First occurred: 11:29:43 AM (7 occurrences)
Last logged: 11:29:43 AM

Unexpected error importing unifiprotect/logbook.py
Unexpected error importing unifiprotect/group.py
Unexpected error importing unifiprotect/media_source.py
Unexpected error importing unifiprotect/system_health.py
Unexpected error importing unifiprotect/cast.py
Traceback (most recent call last):
  File "/usr/src/homeassistant/homeassistant/helpers/integration_platform.py", line 34, in _process
    platform = integration.get_platform(platform_name)
  File "/usr/src/homeassistant/homeassistant/loader.py", line 566, in get_platform
    cache[full_name] = self._import_platform(platform_name)
  File "/usr/src/homeassistant/homeassistant/loader.py", line 583, in _import_platform
    return importlib.import_module(f"{self.pkg_path}.{platform_name}")
  File "/usr/local/lib/python3.9/importlib/__init__.py", line 127, in import_module
    return _bootstrap._gcd_import(name[level:], package, level)
  File "<frozen importlib._bootstrap>", line 1030, in _gcd_import
  File "<frozen importlib._bootstrap>", line 1007, in _find_and_load
  File "<frozen importlib._bootstrap>", line 984, in _find_and_load_unlocked
ModuleNotFoundError: No module named 'custom_components.None.recorder'

**Reply by AngellusMortis** [reply]
It looks like something is seriously wrong with your HA install.

* Make sure you do not have anything under `config/custom_components` or `config/deps`
* Try upgrading to 2022.2.5
* If neither of the above work, try restoring from backup
* If none of those work, create [HA core issue](https://github.com/home-assistant/core/issues/new/choose), post on the forums or try the official HA Discord. Something is going very wrong with how HA itself is handling the integration for you.

**Reply by teedubyeah** [reply]
> It looks like something is seriously wrong with your HA install.
> 
> * Make sure you do not have anything under `config/custom_components` or `config/deps`
> * Try upgrading to 2022.2.5
> * If neither of the above work, try restoring from backup
> * If none of those work, create [HA core issue](https://github.com/home-assistant/core/issues/new/choose), post on the forums or try the official HA Discord. Something is going very wrong with how HA itself is handling the integration for you.

Under `config/custom_components` I have a folder named "HAC" and one named "none".
I have nothing under `config/deps`
